spec.md

```diff
@@ -1125,6 +1141,11 @@ message ControllerServiceCapability {
       // with the snapshot_id as the filter to query whether the
       // uploading process is complete or not.
       LIST_SNAPSHOTS = 6;
+      // Plugins supporting efficient volume cloning MAY report this
+      // capability.  Source volume must be managed by the same plugin.
```

Comment:
we only use a single space between sentences.

also, suggest "A source volume..."

Reply:
Sorry, I've been writing too much on RST lately.